docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -375,6 +375,7 @@ Returns identity and transaction information about a confirmed block in the ledg
   "jsonParsed" encoding attempts to use program-specific instruction parsers to return more human-readable and explicit data in the `transaction.message.instructions` list. If "jsonParsed" is requested but a parser cannot be found, the instruction falls back to regular JSON encoding (`accounts`, `data`, and `programIdIndex` fields).
   - (optional) `transactionDetails: <string>` - level of transaction detail to return, either "full", "signatures", or "none". If parameter not provided, the default detail level is "full".
   - (optional) `rewards: bool` - whether to populate the `rewards` array. If parameter not provided, the default includes rewards.
+  - (optional) `votes: bool` - whether to return the simple vote transactions. This option only work with an encoding  `jsonParsed`. If parameter not provided, the default includes simple votes transactions.
```

Comment:
Maybe also note that this parameter will be a noop when `transactionDetails: none`?

Reply:
I have specified the documentation. Is that ok for you ?

Reply:
Yep, it looks good